### PR TITLE
ripe89 news article

### PR DESCRIPTION
**Pull request by tanneberger**

### CODE REVIEW

File: content/news/012.dd-ix-at-ripe89.en.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: DD-IX at RIPE89 in Prague
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 012_dd_ix_at_ripe89.webp
+---
+
+Marcel, Klemens, and Melody from DD-IX visited the RIPE community at this year's autumn RIPE event in Prague
+RIPE event in Prague. It was a great week with lots of fun and interesting conversations about the future of 
+the Internet in Europe. After winning the RIPE Labs article competition with our IXP-from-Scratch series, 
+Marcel Koch had the opportunity to give a lightning talk at the General Assembly and introduce DD-IX and our article series.
```
Comment (MarcelCoding) [inline content/news/012.dd-ix-at-ripe89.en.md:13]: I would remove the "general assembly".

File: content/news/012.dd-ix-at-ripe89.de.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+published: 2024-11-06
+description: Marcel's Lightning Talk und viele gute Erinnerungen
+title: DD-IX auf RIPE89 in Prag
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 012_dd_ix_at_ripe89.webp
+---
+
+Marcel, Klemens und Melody von DD-IX besuchten gemeinsam die RIPE-Community bei der diesjährigen Herbst-RIPE in Prag.
```
Comment (waehlisch) [inline content/news/012.dd-ix-at-ripe89.de.md:10]: Marcel, Klemens und Melody nahmen am 89. Treffen der RIPE Community teil. RIPE89 fand in Prag statt.

File: content/news/012.dd-ix-at-ripe89.de.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Es war eine tolle Woche mit viel Spaß und interessanten Diskussionen über die Zukunft des Internets in Europa. Nachdem wir mit unserer IXP-from-Scratch Serie den RIPE Labs Artikelwettbewerb gewonnen hatten,
```
Comment (waehlisch) [inline content/news/012.dd-ix-at-ripe89.de.md:11]: Es war eine tolle Woche mit vielen interessanten Diskussionen über die Zukunft des Internets in Europa. Unser persönliches Higlight war der Vortrag von Marcel ;), der unsere RIPE Labs Serie "IXP-from-Scratch" vor mehr als 600 Teilnehmern präsentieren durfte. Der Vortrag ist [zum Nachschauen](https://ripe89.ripe.net/archives/video/1542/) verfügbar.

File: content/news/012.dd-ix-at-ripe89.en.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+published: 2024-11-06
+description: Marcel's Lightning Talk and Memorable Experiences
+title: DD-IX at RIPE89 in Prague
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 012_dd_ix_at_ripe89.webp
+---
+
+Marcel, Klemens, and Melody from DD-IX visited the RIPE community at this year's autumn RIPE event in Prague
```
Comment (waehlisch) [inline content/news/012.dd-ix-at-ripe89.en.md:10]: Marcel, Klemens, andMelody participated at the 89th meeting of the RIPE community. RIPE89 took place in Prague.

File: content/news/012.dd-ix-at-ripe89.en.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+RIPE event in Prague. It was a great week with lots of fun and interesting conversations about the future of
```
Comment (waehlisch) [inline content/news/012.dd-ix-at-ripe89.en.md:11]: It was a great week with interesting discussions about the future of the Internet in Europe. Our personal highlight was the lightning talk given by Marcel ;). He presented our RIPE Labs Serie "IXP-from-Scratch" to more than 600 participants. Haven't had the chance to listen to his presentation yet? A recorded version is available on the [RIPE89 web page](https://ripe89.ripe.net/archives/video/1542/).

**Pull request merged**